pkgs/development/python-modules/gcodepy/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "gcodepy";
+  version = "0.1.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "rmeno12";
+    repo = "gcodepy";
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "v${version}";
```
This should automatically fetch the tag.